docs/getting_started.md

```diff
@@ -1,34 +1,30 @@
-#Getting Started with Jaeger
+#Getting Started
 
-We're working hard on making all backend components of Jaeger open
-source. Please watch the GitHub repository
-<https://github.com/uber/jaeger> for updates.
+##All in one Docker image 
+This image, designed for quick local testing, launches the Jaeger ui, collector, query, and agent, with an in memory storage component. 
 
-Jaeger client libraries are already open source and for now can be used
-with Zipkin backend. You can see a HOW-TO example in the blog post
-[Tracing HTTP request latency in Go with
-OpenTracing](https://medium.com/@YuriShkuro/tracing-http-request-latency-in-go-with-opentracing-7cc1282a100a).
+The simplest way to start the all in one docker image is to use the pre-built image published to docker hub.
 
-##Running Jaeger in Docker Container
-
-Coming soon...
+```bash
+docker run -d -p5775:5775/udp -p16686:16686 jaegertracing/all-in-one:latest
```

Comment:
pity it should work, It works for me without problems. Fedora 25 `Docker version 1.12.6, build ae7d637/1.12.6` installed from official fedora repo.

Could somebody briefly explain these ports? And maybe point me to the source code of those processes?
* `16686` - REST query service
* `5775` - collector for zipkin spans? 
* `6832` - collector for new model? 
* `6831` - ?